### ISSUE TITLE
Lambda + Kinesis: Error Forwarding Request

### ISSUE THREAD

**Issue by vigowei** [atlassian/localstack#107]
`ERROR:localstack.mock.generic_proxy:Error forwarding request: [Errno 32] Broken pipe`

Error above is observed when the output of one lambda triggers another lambda. 

To reproduce:
Set up two streams `K1`, `K2`  and two functions `L1`, `L2` such that records going into `K1` trigger `L1`, `L1` outputs records to `K2`, and `K2` triggers `L2`. In my tests I simply set up`L1` to forward every record to `K2`. 

This error blocks producer from putting more records in the stream. 

```
Traceback (most recent call last):
  File "/opt/code/localstack/localstack/mock/generic_proxy.py", line 116, in forward
    self.send_response(response.status_code)
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 406, in send_response
    self.send_header('Server', self.version_string())
  File "/usr/lib/python2.7/BaseHTTPServer.py", line 412, in send_header
    self.wfile.write("%s: %s\r\n" % (keyword, value))
  File "/usr/lib/python2.7/socket.py", line 328, in write
    self.flush()
  File "/usr/lib/python2.7/socket.py", line 307, in flush
    self._sock.sendall(view[write_offset:write_offset+buffer_size])
error: [Errno 32] Broken pipe
```

**Comment by whummer** [comment]
Thanks for reporting @vigowei , this is fixed in commit 1307728dae9dddeeab26dd6d778c0ba431902fb8 . A new integration test has been added to ensure that the chain of Lambda invocation works.

Can you please pull the latest version of the code (and/or Docker image) and give it a try. Would be great if you could confirm that it works. I'm closing this issue, please re-open if the problem persists. Thanks

**Comment by vigowei** [comment]
@whummer tested with some chained functions and worked. Thanks!
